docs/source/plugins.rst

```diff
@@ -14,7 +14,7 @@ Action
 
 * ``normal``: A fork of Ansible `normal.py`_ action plugin that is modified to allow a conditional shebang line in REXX modules.
```

Comment:
@RadhaV2020 I was going to do this edit after your review but given its here now, we had to act on a JIRA that altered our implementation of the forked normal.py, please make these edits for me.

"* ``normal``: A fork of Ansible `normal.py`_ action plugin that is modified to allow a conditional shebang line in REXX modules." 
TO:

* ``zos_ping``: A fork of Ansible `normal.py`_ action plugin that is modified to allow a conditional shebang line in REXX modules.

Reply:
ack- will update